www/css/style.css

```diff
@@ -965,8 +1021,8 @@ button.button.back-button.buttons.button-clear.header-item {
 .stop-time-tag-lower {
```

Comment:
where is this used?

Reply:
It's not used I don't believe.

Reply:
@sebastianbarry this is used in both lists around the `differentCommon` function. Let's document that and keep it for now, but we should remove it later (potentially).

Reply:
Sounds good, do you have a specific place I should document it? For now I will just make a comment like so:

```css
.stop-time-tag-lower { /* this is used in both lists (infinite_scroll_list.js & list.js) around the differentCommon function */
        box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
        text-align: center;
        background-color: #0088ce;
        color: white;
        width: 16%;
        height: 16px;
        border-radius: 8px;
        font-size: 9px;
        position: absolute;
        left: 1%;
        top: 310px;
        line-height: 16px;
}
```